README.rst

```diff
@@ -18,15 +18,32 @@ Installation
 Usage
 =====
 
-::
-
-    >>> import axelrod_fortran as axlf
-    >>> import axelrod as axl
-    >>> p1 = axlf.Player('k31r')
-    >>> p2 = axlf.Player('k33r')
-    >>> match = axl.Match((p1, p2), turns=5)
-    >>> match.play()
-    [(C, C), (C, C), (C, D), (C, D), (C, C)]
+Running a match:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> p1 = axlf.Player('k31r')
+   >>> p2 = axlf.Player('k33r')
+   >>> match = axl.Match((p1, p2), turns=5)
+   >>> match.play()
+   [(C, C), (C, C), (C, D), (C, D), (C, C)]
+
+Running an instance of Axelrod's second tournament:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> players = [axlf.Player(name) for name in axlf.characteristics.keys()
+   >>>            if axlf.characteristics[name]["original_rank"] is not None]
+   >>> print(len(players), "players")
```

Comment:
Woops, this would fail the doctes as no output is shown. Want to write the output or just delete this line?